engine-rest/engine-rest/src/main/java/org/camunda/bpm/engine/rest/dto/task/TaskQueryDto.java

```diff
@@ -345,6 +346,11 @@ public void setCandidateGroupExpression(String candidateGroupExpression) {
     this.candidateGroupExpression = candidateGroupExpression;
   }
 
+  @CamundaQueryParam("candidateGroupLike")
```

Comment:
Please add this new parameter to the [task-query-params.ftl](https://github.com/camunda/camunda-bpm-platform/blob/8de6e2d4da5bbdcc3e1eb75d03b08e97b6fbeb9d/engine-rest/engine-rest-openapi/src/main/templates/lib/commons/task-query-params.ftl#L198) file to update the rest api documentation.

Reply:
Also for the POST requests, in [this file](https://github.com/camunda/camunda-bpm-platform/blob/79ff55d871f4983bb4c178e14855ba80ffecaea8/engine-rest/engine-rest-openapi/src/main/templates/models/org/camunda/bpm/engine/rest/dto/runtime/task/TaskQueryDto.ftl#L207).

Reply:
Fixed